ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.16.yaml

```diff
@@ -347,6 +347,12 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-nutanix-ipi-zones-customized-resource
+- as: nutanix-ipi-invaid-fields-f28
+  cluster: build01
+  cron: 2 20 16 * *
+  steps:
+    cluster_profile: nutanix-qe-zone
```

Comment:
Shall we use the zone cluster profile to run the testing ?

Reply:
Okay, sounds like cucushift-installer-rehearse-nutanix-ipi-invaid-fields includes some zone setting check, if that, I think we should rename the install workflow to indicated it is a zonal install workflow.